code/__HELPERS/areas.dm

```diff
@@ -87,15 +87,20 @@ GLOBAL_LIST_INIT(typecache_powerfailure_safe_areas, typecacheof(/area/station/en
 	var/static/list/blacklisted_areas = typecacheof(list(
 		/area/space,
 		))
+
+	var/error = null
```

Comment:
```suggestion
	var/error = ""
```

doing it as "null" isn't exactly useful since this is going to be mutated into a string anyways, so it should be typed as a string like such.